script.js

```diff
@@ -1,75 +1,96 @@
+// const rock = document.querySelector("#rock");
+// const paper = document.querySelector("#paper");
+// const scissors = document.querySelector("#scissors");
+const buttons = document.querySelectorAll("button");
+const playerScore = document.querySelector(".playerScore");
+const cpuScore = document.querySelector(".cpuScore");
+const displayMessage = document.querySelector("#displayMessage");
+
+let computerSelection;
+let humanSelection;
+let humanScore = 0;
+let computerScore = 0;
+
+playerScore.textContent = humanScore;
+cpuScore.textContent = computerScore;
+
+buttons.forEach((button) => {
+    button.addEventListener("click", (e) => {
+        computerSelection = getComputerChoice();
+        humanSelection = e.target.getAttribute("id");
+        playGame();
+    });
+});
+
+// rock.addEventListener("click", (e) => {
+//     computerSelection = getComputerChoice();
+//     humanSelection = e.target.getAttribute("id");
+//     playGame();
+// })
+// paper.addEventListener("click", (e) => {
+//     computerSelection = getComputerChoice();
+//     humanSelection = e.target.getAttribute("id");
+//     playGame();
+// })
+// scissors.addEventListener("click", (e) => {
+//     computerSelection = getComputerChoice();
+//     humanSelection = e.target.getAttribute("id");
+//     playGame();
+// })
+
 function getComputerChoice() {
-    let cpuChoice;
+
     let choice = Math.floor(Math.random() * 3) + 1;
     
     if (choice === 1) {
-        cpuChoice = "ROCK";
+        return "rock";
     }
     else if (choice === 2) {
-        cpuChoice = "PAPER";
+        return "paper";
     }
     else {
-        cpuChoice = "SCISSORS";
+        return "scissors";
     }
-
-    return cpuChoice;
 }
 
-function getHumanChoice() {
-    let humanChoice = prompt("Rock, Paper, or Scissors?");
-    humanChoice = humanChoice.toUpperCase();
-
-    if(humanChoice === "ROCK" || humanChoice === "PAPER" || humanChoice === "SCISSORS"){
-        return humanChoice;
-    }
-    else {
-        humanChoice = "ROCK";
-        alert("You no try. You don choose rock be that")
-        return humanChoice;
-    }
+function updateScore() {
+    playerScore.textContent = humanScore;
+    cpuScore.textContent = computerScore;
 }
 
 function playGame() {
-    let humanScore = 0;
-    let cpuScore = 0;
-
```

Comment:
These variables are re-declared within the playGame function, overriding the global humanScore and computerScore. I feel it can cause the scores to reset every time playGame is called. Wouldn't it be better to remove the let declaration inside playGame function and use the global declaration?

Reply:
These codes you are referring to are codes that I have already removed. That's probably why they are in red